assets/base.css

```diff
@@ -264,8 +264,14 @@ h5,
   word-break: break-word;
 }
 
+.hxxl {
+  font-size: clamp(calc(var(--font-heading-scale) * 5.6rem), 14vw, calc(var(--font-heading-scale) * 7.2rem));
+  line-height: 1.1;
```

Comment:
yeah I think the difference too with the `hxl` is that the division was happening on `1.1` while on there it happens on the `0.3` value because it's defined like this: `1 + 0.3 / ....`